kolibri/core/assets/src/vue/assessment-wrapper/index.vue

```diff
@@ -107,8 +107,9 @@ oriented data synchronization.
       },
     },
     computed: {
-      notDeviceOwner() {
-        return !this.userkind.includes(UserKinds.SUPERUSER);
+      isFacilityUser() {
```

Comment:
it does not include anonymous users
`!(this.userkind.includes(UserKinds.SUPERUSER) || this.userkind.includes(UserKinds.ANONYMOUS));`